README.md

```diff
@@ -62,6 +62,20 @@ user:                         # you can have multiple profiles for different pro
 
 `dbt clean` to clear compiled files
 
+#### docker
+
+an alternate means of using dbt is with the docker image jthandy/dbt. if you already have docker installed on your system, run with:
+```docker run -v ${PWD}:/dbt -v ~/.dbt:/root/.dbt jthandy/dbt /bin/bash -c "[type your command here]"
+this can be run from any dbt project directory. it relies on the same configuration setup outlined above.
+
+on linux and osx hosts, running this can be streamlined by including the following function in your bash_profile:
+```
+function dbt() {
```

Comment:
@3mei I don't think this actually works... i tested it out on a folder with spaces and got an error. I think you generally have to escape spaces as opposed to quote them in bash, but I'm not an expert on that...regardless it didn't work. 

I'm not sure that we should spend a lot of time troubleshooting a helper script though--would rather just push this.